test/checkouts_test.rb

```diff
@@ -1,9 +1,19 @@
 require 'test_helper'
 
 class CheckoutsTest < Test::Unit::TestCase
-  test "get all should get all orders" do
-    fake 'checkouts', :method => :get, :status => 200, :body => load_fixture('checkouts')
+  test "get all checkouts indexed by token" do
+    fake 'checkouts', method: :get, status: 200, body: load_fixture('checkouts')
     checkout = ShopifyAPI::Checkout.all
-    assert_equal 450789469, checkout.first.id
+    assert_equal '2a1ace52255252df566af0faaedfbfa7', checkout.first.id
```

Comment:
shouldn't this be `checkout.first.token`? Or is `id` aliased to the primary key? Either way, I think `token` is more explicit

Reply:
`ActiveResource` objects use `id` variable as primary key, mapped by default to an `id` attribute of the mapped object. But in this case, I overrode the property to be mapped to `token` attribute: https://github.com/Shopify/shopify_api/pull/462/files#diff-7d86954f571e1b61789ade0a470543b1R5.

I'd prefer keep using `id` when manipulating `ActiveResource` objects, as I think it is more consistent, even though it might be less explicit.

Reply:
Ok, makes sense 👍